sig-testing/README.md

```diff
@@ -65,6 +57,10 @@ Kubernetes IN Docker. Run Kubernetes test clusters on your local machine using D
   - sigs.k8s.io/kind bi-weekly meeting: [Mondays at 11:00 PT (Pacific Time)](https://zoom.us/j/960461819) (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
     - [Meeting notes and Agenda](https://docs.google.com/document/d/1b9Ppm7ZT_tMWRs5Ph1zGJJKb5nF9c3ZHzMwg1olJIrc/edit).
     - [Meeting recordings](https://bit.ly/k8s-sig-testing-videos).
+### kubetest2
```

Comment:
pretty sure you mispelled "kates" cc @amwat

Reply:
leave a vote on the doc 😬 
shouldn't block this PR, will rename it here if we finalize on the rename.
Will definitely finalize the name before requesting for a new repo though 👍 

Thanks!

/lgtm
/hold